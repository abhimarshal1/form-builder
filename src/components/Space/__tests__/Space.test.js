import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Components
import { Space } from '..';

const renderer = new ShallowRenderer();

describe('Space', () => {
  test('shallow render & match the snapshot', () => {
    renderer.render(<Space flexDirection="Column" />);

    const output = renderer.getRenderOutput();
    expect(output).toMatchSnapshot();
  });
});
