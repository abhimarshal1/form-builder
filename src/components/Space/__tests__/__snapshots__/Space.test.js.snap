// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Space shallow render & match the snapshot 1`] = `
<div
  className="styled__Space-sc-kxtek7-0 kMHmik"
/>
`;
